pom.xml

```diff
@@ -465,7 +464,7 @@
                         <threadCount>${air.test.thread-count}</threadCount>
                         <argLine>
                             ${argLine}
-                            -Duser.timezone=${air.test.timezone}
```

Comment:
Yes. IntelliJ needs it in system properties to pass it as an argument, and Surefire warns if you have it both places. I tested this and it seems to work fine. I'm actually not sure why we added it here in the first place.